### ISSUE TITLE
printf formatting specifiers; no documentation of argument-provided padding.

### ISSUE THREAD

**Issue by MorganLOCode** [php/doc-en#2423]
Since PHP 8.0, a [width] specifier of `*` has been allowed, which is replaced by the corresponding next argument used (or however that should be phrased).
```php
printf("[%'#*s]\n", 10, 'monkey');
```
being equivalent to
```php
printf("[%'#10s]\n", 'monkey');
```
results in
```text
[####monkey]
```
but this is not yet documented. The description for "width" is merely that it is an integer specifying a minimum width, not that it can be parameterised in the above fashion.

Ditto for the other *printf functions.
http://www.php.net/printf
http://www.php.net/sprintf
http://www.php.net/fprintf
http://www.php.net/vprintf
http://www.php.net/vsprintf
http://www.php.net/vfprintf

**Comment by MorganLOCode** [comment]
Just checked, and confirmed that [precision] can also be `*`d.

---

**Width**
Either an integer that says how many characters (minimum) this conversion should result in, or `*`. If `*` is used, then the width is supplied as an additional integer value preceding the one formatted by the specifier.

**Comment by Girgias** [comment]
Fixed via #2474
